config/metadata/physical_description_metadata.yml

```
@@ -13,3 +13,4 @@ illustrations_note_tesim: 'Illustrations note'
 condition_note_tesim: 'Condition note'
 binding_note_tesim: 'Binding note'
 inscription_tesim: 'Inscription'
+opac_url_tesim: 'Opac url'
```

```suggestion
opac_url_ssi: 'Opac url'
```
Does this field go in more than one yml file

---

looks good.